### ISSUE TITLE
A/B Test New Onboarding Cards

### ISSUE THREAD

**Issue by athomasmoz** [mozilla-mobile/firefox-ios#6434]
Vision

An A/B test is running to determine if the newly design onboarding cards resonate better with users

**Acceptance Criteria**

* Leanplum A/B test set up for onboarding where,
  * A displays the 2 updated onboarding cards, and is shown to X% of the audience
  * B displays the 2 existing onboarding cards, and is shown to Y% of the audience
* Updated onboarding cards match the [mocks & content](https://app.abstract.com/share/fb3066fc-2401-48a1-951d-609eb81c3913)
* Data science has signed off on the split & experiment set up
* Ticket created for DS to analyze the results of the A/B test based on the [success criteria](https://docs.google.com/document/d/1diKGUoDdM7uq4IUZgbwtEbgcM-GouP0A0AKHnuPulkY/edit#heading=h.79ad2m9je1va)

**Comment by athomasmoz** [comment]
Bumping up from 3 to 5 to give time for refactoring/cleanup in the area

**Comment by nbhasin2** [comment]
FYI - As a part of this feature I have asked LP to enable getting name of the A/B test in the SDK.

```
let variants = Leanplum.variants()
print("Variants: \(variants)")

Output:
Variants: Optional([{
    abTestId = 5299325898719232;
    abTestName = "Show Onboarding A/A Test";
    id = 6019555641917440;
    name = "Variant 1";
}])
```

**Comment by apbitner** [comment]
We've received an updated illustration for the second card from the UX Systems team. It's been updated in the spec, but I'm attaching the SVG here as well.

[illustration-syncing-devices.svg.zip](https://github.com/mozilla-mobile/firefox-ios/files/4611658/illustration-syncing-devices.svg.zip)

**Comment by nbhasin2** [comment]
@apbitner Marking it as Needs UX as the format we need is PDF vector image

**Comment by apbitner** [comment]
I have an updated illustration for you to try, let me know if you still have issues.

[illustration-syncing-devices.pdf](https://github.com/mozilla-mobile/firefox-ios/files/4617477/illustration-syncing-devices.pdf)

**Comment by nbhasin2** [comment]
How to test the views (only)
Added a option to toggle between Default Intro View Controller and Version 2 of it in the hidden settings

- Enable hidden settings
    - Tap on Debug: Toggle onboarding type
    - Tap on Show Tour in the settings
    
Repeat above steps to switch between default and version 2 of the intro view controller

**Comment by SimonBasca** [comment]
Verified fix on 26.0 (18036)
